### PR TITLE
Add ValueOrDefault option to Input and TextArea.

### PR DESCRIPTION
**Pull request by marcpiechura**
I've copied the implementation from

https://github.com/elmish/react/blob/389ff15eec694322a2f66354b49361196c128529/docs/content/react.fsx#L25

### CODE REVIEW

File: src/Fulma/Elements/Form/Input.fs [hunk 1]
```diff
@@ -60,6 +60,9 @@ module Input =
         | Value of string
         /// Set `DefaultValue` HTMLAttr
         | DefaultValue of string
+        /// `Ref` callback that sets the value of an input textbox after DOM element is created.
+        /// Can be used override input box value.
```
Comment (MangelMaxime) [inline src/Fulma/Elements/Form/Input.fs]: We can probably remove this line

File: src/Fulma/Elements/Form/Textarea.fs [hunk 2]
```diff
@@ -45,6 +45,9 @@ module Textarea =
         | Value of string
         /// Set `DefaultValue` HTMLAttr
         | DefaultValue of string
+        /// `Ref` callback that sets the value of an input textbox after DOM element is created.
+        /// Can be used override input box value.
```
Comment (MangelMaxime) [inline src/Fulma/Elements/Form/Textarea.fs]: We can probably remove this line

**Comment by marcpiechura** [comment]
@MangelMaxime  addressed your remarks

**Comment by MangelMaxime** [comment]
Thank you :)

**Pull request merged**